packages/mdl-button/index.scss

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-button {
```

Comment:
Spoke with one of the motion designers, and we came up with this as a fallback: http://codepen.io/traviskaufman/pen/rLKppa

It's very similar to yours but it uses pseudo-elements to handle the shades for active and focused states, and adds a subtle transition to make it feel more material. We can use this as a simple implementation and then create an issue to add the ripple to the button at a later point. WDYT?